WordPress/Classes/ViewRelated/Site Creation/WebAddress/WebAddressWizardContent.swift

```diff
@@ -359,101 +353,7 @@ extension WebAddressWizardContent: NetworkStatusDelegate {
 
 extension WebAddressWizardContent: UITextFieldDelegate {
     func textFieldShouldClear(_ textField: UITextField) -> Bool {
-        resetTableOffsetIfNeeded()
+        tableViewOffsetCoordinator?.resetTableOffsetIfNeeded()
         return true
     }
 }
-
-
-// MARK: - Keyboard management
-
-private extension WebAddressWizardContent {
-    struct Constants {
-        static let bottomMargin             = CGFloat(0)
-        static let headerAnimationDuration  = Double(0.25)  // matches current system keyboard transition duration
-        static let topMargin                = CGFloat(36)
-    }
-
-    func adjustTableOffsetIfNeeded(_ animationDuration: Double = Constants.headerAnimationDuration) {
-        guard WPDeviceIdentification.isiPhone(), bottomConstraintConstant > 0, tableViewHasBeenAdjusted == false else {
-            return
-        }
-
-        bottomConstraint.constant = bottomConstraintConstant
-        view.setNeedsUpdateConstraints()
-
-        let targetInsets: UIEdgeInsets
-        if let header = table.tableHeaderView as? TitleSubtitleTextfieldHeader {
-            let textfieldFrame = header.textField.frame
-            targetInsets = UIEdgeInsets(top: (-1 * textfieldFrame.origin.y) + Constants.topMargin, left: 0.0, bottom: bottomConstraintConstant, right: 0.0)
-        } else {
-            targetInsets = .zero
-        }
-
-        UIView.animate(withDuration: animationDuration, delay: 0, options: .beginFromCurrentState, animations: { [weak self] in
-            guard let self = self else {
-                return
-            }
-
-            self.view.layoutIfNeeded()
-            self.table.contentInset = targetInsets
-            self.table.scrollIndicatorInsets = targetInsets
-            if let header = self.table.tableHeaderView as? TitleSubtitleTextfieldHeader {
-                header.titleSubtitle.alpha = 0.0
-            }
-            }, completion: { [weak self] _ in
-                self?.tableViewHasBeenAdjusted = true
-        })
-    }
-
-    @objc
-    func keyboardWillShow(_ notification: Foundation.Notification) {
-        guard let payload = KeyboardInfo(notification) else {
-            return
-        }
-
-        let keyboardScreenFrame = payload.frameEnd
-        let convertedKeyboardFrame = view.convert(keyboardScreenFrame, from: nil)
-
-        var adjustedKeyboardHeight = convertedKeyboardFrame.height
-        if #available(iOS 11.0, *) {
```

Comment:
This method seems to be the only tangible difference between the new TableViewOffsetCoordinator and what was in the two classes.

There is no longer a specific check for iOS 11+. Is that ok? I imagine this check was put here on purpose before.

Reply:
Good question @kwonye. 

The "safe area" was introduced in iOS 11, but we don't _actually_ need to consider it here, as the keyboard height accounts for it.

The original implementation was adjusting the bottom constraint of the table view, but we only need to adjust the `contentInset`.